source/dubregistry/dbcontroller.d

```diff
@@ -369,6 +370,24 @@ class DbController {
 		foreach (ref pkg; pkgs)
 			pkg.textScore = (pkg.textScore - minMaxTS[0]) * scale + DbPackageStats.minScore;
 
+		// Also search for substring matches in package name using regex
+		auto regexPkgs = m_packages
+			.find(["name": ["$regex": query, "$options": "i"]])
```

Comment:
missing regex escape + max length, this could otherwise lead to a DoS

You added the search to the full-text-search, is that not sufficient for the results?

Reply:
Thanks for the feedback. Element took my PR fixed the regex issue you mentioned and added some more, so I will close this.

Reply:
um I don't think there is a PR opened for this yet though?